monri/src/main/java/com/monri/android/activity/ConfirmPaymentResponseCallback.java

```diff
@@ -23,7 +23,7 @@
  * Created by jasminsuljic on 2019-12-09.
  * MonriAndroid
  */
-final class ConfirmPaymentResponseCallback implements ResultCallback<ConfirmPaymentResponse> {
+public final class ConfirmPaymentResponseCallback implements ResultCallback<ConfirmPaymentResponse> {
```

Comment:
@adnanomerovic Why is this public?

Reply:
My mistake, I did something in test but forgot to put it back. I'll fix that in next commit.